src/add_remove.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Maybe you can find a way to move the function ```fillOutList``` from ```index.js``` into ```add_remove.js```, this is creating a cycle import, try not to disable linters unless extremely necessary.